kubernetes-itests/src/test/java/io/fabric8/kubernetes/CronJobIT.java

```diff
@@ -53,30 +53,30 @@ public static void init() {
 
   @Test
   public void load() {
-    CronJob aCronJob = client.batch().cronjobs().load(getClass().getResourceAsStream("/test-cronjob.yml")).get();
+    CronJob aCronJob = client.batch().v1().cronjobs().load(getClass().getResourceAsStream("/test-cronjob.yml")).get();
```

Comment:
`client.batch().cronjobs()` returns MixedOperation for `io.fabric8.kubernetes.api.model.batch.v1beta1` 

`client.batch().v1().cronjobs()` returns MixedOperation for `io.fabric8.kubernetes.api.model.batch.v1`

Maybe we should have a separate test class named V1CronJobIT for v1 CronJob test. We run our integration tests on different Kubernetes versions. We can add a check in that test to only run for Kubernetes versions > 1.21.0 . I'm not sure if we should do it as a part of this PR or in a separate issue.
```
  @ClassRule
  public static final AssumingK8sVersionAtLeast assumingK8sVersion =
    new AssumingK8sVersionAtLeast("1", "21");
```